src/System.Composition.Hosting/src/System/Composition/Hosting/Core/CompositionDependency.cs

```diff
@@ -140,7 +170,10 @@ public override string ToString()
 
         internal void DescribeError(StringBuilder message)
         {
-            Assumes.IsTrue(IsError, "Dependency is not in an error state.");
+            if(!IsError)
+            {
+                throw new Exception(String.Format(CultureInfo.CurrentCulture, SR.Dependency_Not_In_Error_State));
```

Comment:
no string.format required here. you would only need string.format with the current culture if you have formatting directives in the resource string like a {0:D} or something. that's not the case here.